app/react-native/src/server/config/webpack.config.js

```diff
@@ -4,8 +4,9 @@ import { getEnvironment } from 'universal-dotenv';
 import Dotenv from 'dotenv-webpack';
 import WatchMissingNodeModulesPlugin from 'react-dev-utils/WatchMissingNodeModulesPlugin';
 import CaseSensitivePathsPlugin from 'case-sensitive-paths-webpack-plugin';
-import HtmlWebpackPlugin from 'html-webpack-plugin';
-import { indexHtmlPath } from '@storybook/core/server';
+import GeneratePagePlugin from 'generate-page-webpack-plugin';
+
+import { getManagerHeadHtml } from '@storybook/core/dist/server/utils';
```

Comment:
I think it's better to be exported from `core/server`

Reply:
right